### PR TITLE
Feature/lvjav442 45

### PR DESCRIPTION
**Pull request by myzonchuk**

### CODE REVIEW

File: src/main/java/com/softserve/ita/java442/cityDonut/controller/RegistrationController.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/v1/registration")
+public class RegistrationController {
+
+    @Autowired
+    UserServiceImpl userServiceimpl;
```
Comment (NazarRyvak) [inline src/main/java/com/softserve/ita/java442/cityDonut/controller/RegistrationController.java]: private 19row

File: src/main/java/com/softserve/ita/java442/cityDonut/controller/RegistrationController.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+    @PostMapping("/")
+    public ResponseEntity<UserRegistrationDto> registration(@RequestBody UserRegistrationDto dto) {
+        System.out.println(dto);
```
Comment (NazarRyvak) [inline src/main/java/com/softserve/ita/java442/cityDonut/controller/RegistrationController.java]: delete sout

File: src/main/java/com/softserve/ita/java442/cityDonut/exception/BadEmailException.java [hunk 3]
```diff
@@ -0,0 +1,9 @@
+package com.softserve.ita.java442.cityDonut.exception;
+
+public class BadEmailException extends RuntimeException {
+
+    public BadEmailException(String message) {
+        super(message);
+
```
Comment (NazarRyvak) [inline src/main/java/com/softserve/ita/java442/cityDonut/exception/BadEmailException.java]: delete row

File: src/main/java/com/softserve/ita/java442/cityDonut/exception/IncorrectPasswordException.java [hunk 4]
```diff
@@ -4,5 +4,6 @@
 
     public IncorrectPasswordException(String message) {
         super(message);
+
```
Comment (NazarRyvak) [inline src/main/java/com/softserve/ita/java442/cityDonut/exception/IncorrectPasswordException.java]: delete row

File: src/main/java/com/softserve/ita/java442/cityDonut/exception/InvalidEmailException.java [hunk 5]
```diff
@@ -0,0 +1,9 @@
+package com.softserve.ita.java442.cityDonut.exception;
+
+public class InvalidEmailException extends RuntimeException {
+
+    public InvalidEmailException(String message) {
+        super(message);
+
```
Comment (NazarRyvak) [inline src/main/java/com/softserve/ita/java442/cityDonut/exception/InvalidEmailException.java]: delete row

**Review by NazarRyvak — APPROVED** [review]

**Review by martaprotsak — APPROVED** [review]

**Pull request merged**